### ISSUE TITLE
Teleport Destination drawing order is incorrect when building with URP

### ISSUE THREAD

**Issue by shiena** [ValveSoftware/steamvr_unity_plugin#875]
Environment
- Unity2019.4.16f1
- SteamVR Unity Plugin v2.6.1
- URP 7.3.1

Probably due to `Assets/SteamVR/InteractionSystem/Teleport/Shaders/Highlight.shader`.
For some reason, it is a normal drawing order in Unity Editor.

### URP
![image](https://user-images.githubusercontent.com/1295639/101980590-c825eb80-3ca9-11eb-923c-1db224d76b5e.png)
![image](https://user-images.githubusercontent.com/1295639/101980595-cfe59000-3ca9-11eb-9bda-dea5083b3ae1.png)
### Builtin
![image](https://user-images.githubusercontent.com/1295639/101980606-df64d900-3ca9-11eb-9c53-6d103c9e09c3.png)
![image](https://user-images.githubusercontent.com/1295639/101980608-e7bd1400-3ca9-11eb-98c8-82b6b8c70681.png)

**Comment by shiena** [comment]
It may be the same as problem https://github.com/ValveSoftware/steamvr_unity_plugin/issues/422

**Comment by shiena** [comment]
I found a solution.
Replaces the TeleportXXX material in the Teleporting prefab with the URP TeleportXXX material.
![image](https://user-images.githubusercontent.com/1295639/101985940-42686700-3cce-11eb-859c-4c7c82fd7936.png)

**Comment by shiena** [comment]
I found the cause of the incorrect drawing of the built application.
Teleporting prefab replaces the material with TeleportURPHelper.cs. However, since the `#if UNITY_UDP && UNITY_EDITOR` directive is specified, it cannot be replaced when building.
https://github.com/ValveSoftware/steamvr_unity_plugin/blob/master/Assets/SteamVR/InteractionSystem/Teleport/Scripts/TeleportURPHelper.cs#L11

**Comment by zite** [comment]
yeah, the intention is to get this changed in editor so it doesn't need to be done at runtime. Are you finding this does not get run / saved in editor?

**Comment by kannankans52** [comment]
Teleport Destination drawing order is incorrect when building with URP
please find Above issue and please instruct me how to fix this issue

**Comment by kannankans52** [comment]
> Environment
> 
> * Unity2019.4.16f1
> * SteamVR Unity Plugin v2.6.1
> * URP 7.3.1
> 
> Probably due to `Assets/SteamVR/InteractionSystem/Teleport/Shaders/Highlight.shader`. For some reason, it is a normal drawing order in Unity Editor.
> 
> ### URP
> ![image](https://user-images.githubusercontent.com/1295639/101980590-c825eb80-3ca9-11eb-923c-1db224d76b5e.png) ![image](https://user-images.githubusercontent.com/1295639/101980595-cfe59000-3ca9-11eb-9bda-dea5083b3ae1.png)
> 
> ### Builtin
> ![image](https://user-images.githubusercontent.com/1295639/101980606-df64d900-3ca9-11eb-9c53-6d103c9e09c3.png) ![image](https://user-images.githubusercontent.com/1295639/101980608-e7bd1400-3ca9-11eb-98c8-82b6b8c70681.png)

Teleport Destination drawing order is incorrect when building with URP
please find Above issue and please instruct me how to fix this issue

**Comment by kannankans52** [comment]
> > Replying to [#875 (comment)](https://github.com/ValveSoftware/steamvr_unity_plugin/issues/875#issue-763516950)
> 
> Teleport Destination drawing order is incorrect when building with URP please find Above issue and please instruct me how to fix this issue

Thanks for your reply,

I am using URP Material for that teleporting with 3D plane it's working fine. but when I try to teleport inside any gameobject(Cube..) the **Destination Reticle** and **Raycast** but is not visible. so i try with normal material which means without URP material, now in this case working fine with gameobject(Cube) but it's not visible in 3d plane.

Please fix this issue ASAP.
